game_loop_entity.py

```diff
@@ -85,3 +129,7 @@ def run(self):
 
```

Comment:
`get_opponent_move_from_library` returns a `ChessMove`, not 2 `ChessPosition`s. Change this to read:

```
opp_move = self.get_opponent_move_from_library()
self.show_opponent_move(opp_move.init_pos, opp_move.final_pos)
```